test/local-ci.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+test/docker_vunit.sh && node dist/test/test.js && npx jest && npx eslint --ext .ts --fix .
```

Comment:
```suggestion
test/docker_vunit.sh && npm run test && npm run lint```

Reply:
This is just for own dummy lucal testing and this way tsc gets not run (because I am watching for it anyways.)

Reply:
Also npm run lint does not fix